enterprise/frontend/src/embedding-sdk/cli/run.ts

```diff
@@ -42,14 +43,20 @@ export const CLI_STEPS = [
   { id: "addDatabaseConnection", executeStep: addDatabaseConnectionStep },
   { id: "pickDatabaseTables", executeStep: pickDatabaseTables },
   { id: "createModelsAndXrays", executeStep: createModelsAndXrays },
-  {
-    id: "generateReactComponentFiles",
-    executeStep: generateReactComponentFiles,
-  },
   { id: "setupLicense", executeStep: setupLicense },
+
+  // The following steps require the license to be defined first.
   { id: "setupEmbeddingSettings", executeStep: setupEmbeddingSettings },
   { id: "askForTenancyColumns", executeStep: askForTenancyColumns },
   { id: "setupPermissions", executeStep: setupPermissions },
+  {
+    id: "generateReactComponentFiles",
```

Comment:
@WiNloSt I just discussed with Alberto on this yesterday, I'll update the prompt to be more obvious that the license prompt is skippable. As for moving the step after the token, it would still generate the components fine as the license step is optional. We can still proceed to the next step even if there is no license.

Reply:
Cool then.